integration-tests/src/test/kotlin/com/google/devtools/ksp/test/KMPImplementedIT.kt

```diff
@@ -57,9 +57,16 @@ class KMPImplementedIT {
         )
 
         verify(
-            "workload/build/libs/workload-js-1.0-SNAPSHOT.jar",
+            "workload/build/libs/workload-jslegacy-1.0-SNAPSHOT.jar",
             listOf(
-                "playground-workload.js"
+                "playground-workload-js-legacy.js"
+            )
+        )
+
+        verify(
+            "workload/build/libs/workload-jsir-1.0-SNAPSHOT.klib",
+            listOf(
+                "default/ir/types.knt"
```

Comment:
I found lots of files in the generated `klib` file, is this a special file we are interested in? I somehow reached to same test but I just passed an empty list for content to check.

Reply:
I chose this somewhat looks stable file only to make sure the output isn't empty. Future compiler bumps could break it but it'll be caught.